internal/testrunner/testrunner.go

```diff
@@ -253,6 +255,20 @@ func Run(testType TestType, options TestOptions) ([]TestResult, error) {
 		return runner.TearDown()
 	}
 
+	// Handle signals, incl. ctrl+c
+	ch := make(chan os.Signal)
+	signal.Notify(ch, os.Interrupt, syscall.SIGTERM)
+	go func() {
+		<-ch
+		logger.Info("Signal caught!")
+
+		err := tearDown()
+		if err != nil {
+			logger.Errorf("can't tear down the test runner: %v", err)
+		}
+		os.Exit(1)
+	}()
+
```

Comment:
This definitely works so I don't want to block this PR.

But in case some day `elastic-package` gets more long-running sub-commands, we might want to consider moving this logic to a higher level (not sure where exactly yet) so that a SIGTERM or SIGINT could invoke tear down, if implemented, on that sub command.

BTW, we already have another long-running sub command: `elastic-package stack up` (without the `-d` flag). But it looks like SIGINT is already being handled by the underlying lib.

Reply:
I can improve on this in follow-up issues, but this particular one I wanted to focus on improving developer experience of a person writing system tests (and rerunning tests multiple times).

Reply:
++ to follow up PRs, as needed.